.gitmodules

```diff
@@ -9,7 +9,7 @@
 	url = https://github.com/zephyrproject-rtos/newlib-cygwin.git
 [submodule "gcc"]
 	path = gcc
-	url = https://github.com/zephyrproject-rtos/gcc.git
+	url = https://github.com/thaoluonguw/gcc.git
```

Comment:
we should get this merged and thus picked automatically: https://github.com/zephyrproject-rtos/gcc/pull/28
FYI @stephanosio

Reply:
@tejlmand, https://github.com/zephyrproject-rtos/gcc/pull/28 is merged. I updated submodule to reference the merged commit in gcc. Please help me review for it. Thank you.